SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialStatics.cpp

```diff
@@ -107,8 +107,20 @@ bool USpatialStatics::IsSpatialMultiWorkerEnabled(const UObject* WorldContextObj
 		return SpatialGDKSettings->bOverrideMultiWorker.GetValue();
 	}
 
-	const ASpatialWorldSettings* WorldSettings = Cast<ASpatialWorldSettings>(World->GetWorldSettings());
-	return WorldSettings != nullptr && WorldSettings->IsMultiWorkerEnabledInWorldSettings();
+	ASpatialWorldSettings* WorldSettings = Cast<ASpatialWorldSettings>(World->GetWorldSettings());
```

Comment:
As per the GDK coding guidelines, please more the WorldSettings definition into the if statement.
So it becomes -
`if (ASpatialWorldSettings* WorldSettings = Cast<ASpatialWorldSettings>(World->GetWorldSettings()))`